### DISCUSSION TITLE
Example of Next.js with Okta Auth

### DISCUSSION THREAD

**Discussion by jameswlane** [vercel/next.js#14455] [Help]
I am looking at using Next.js for my next project, after some google searching I have yet to see a example of using Next.js custom server with Okta auth. I have seen a lot of examples for React and Express. But since there is some abstraction using Next.js I am wondering if these examples would still work. 

Has anyone seen examples or articles they could point me to or have had experience with using Next.js with Okta?

**Comment by andrzejchmura** [comment]
I haven't used it myself, but NextAuth.js seems to support Okta integration, as shown [here](https://next-auth.js.org/providers/Okta). Digging through the source code might be helpful.

**Reply by jameswlane** [reply]
Awesome, Thank you @andrzejchmura I will check it out.

**Reply by OlgitH** [reply]
Hi, did you manage to get this working? I am trying to integrate okta with using next-auth but can't get it to work

**Reply by aegiz** [reply]
See this full tutorial here: https://developer.okta.com/blog/2020/11/13/nextjs-typescript
